pkg/logs/internal/parsers/interface.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package parsers supports transforming raw log "lines" into messages with some
+// associated metadata (timestamp, severity, etc.).
+//
+// This parsing comes after "line parsing" (breaking input into multiple lines) and
+// before further processing and aggregation of log messages.
+package parsers
+
+// Message represents a message parsed from a single line of log data
+type Message struct {
```

Comment:
While we are here, what do you think about renaming this as well? 
While it's in a differnt package it is often confusing when browsing the code if you are dealing with a `parsers.Message` or a `message.Message` (compounded by the fact that they share a few feilds). 

I don't have a particualrly good suggestion for a new name. Maybe `LogLine`, `ParsedLine`, or something like that. WDYT?

Reply:
I'll add it to my list.  I think it would be more performant to have this type "go away" and use a common type throughout -- as it is, we're allocating parsers.Message's, then allocating decoder.Message's and copying the fields into them, which isn't very efficient!